src/main/java/com/twilio/sdk/resource/instance/Account.java

```diff
@@ -846,6 +847,181 @@ public UsageRecordList getUsageRecords() {
 	}
 
 	/**
+	 * Returns multiple Usage Records for each usage category,
+	 * each representing usage over a daily time-interval
+	 *
+	 * @return list of Usage Records over a daily time-interval
+	 */
+	public UsageRecordList getDailyUsageRecords() {
```

Comment:
ok sounds good!

I'll admit to this being a total nitpick, but this still looks like a lot of copy pasted code. Why not have one generic method  (this one can be `private` if we want our users to use these) that all 14 of these call into?